test/clojure/core_test/nth.cljc

```diff
@@ -16,4 +16,9 @@
     (is (thrown? #?(:cljs :default, :clj Exception) (nth [0 1 2] -1)))
     (is (thrown? #?(:cljs :default, :clj Exception) (nth [0 1 2] 10)))
     (is (thrown? #?(:cljs :default, :clj Exception) (nth [0 1 2] nil)))
-    (is (thrown? #?(:cljs :default, :clj Exception) (nth nil nil)))))
+    (is (thrown? #?(:cljs :default, :clj Exception) (nth nil nil)))
+
+    ;; `nth` accepts a default argument
+    (is (= :default (nth nil 0 :default)))
+    (is (= :default (nth [0] 1 :default)))
+    (is (= :default (nth [0 1] 2 :default)))))
```

Comment:
Let's also get a test for a negative number in here, please. Like we have on line 16, except it returns the default.

Reply:
Done